test/performance/3-multiple-blobs-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+echo 'azmi - build executable'
+dotnet build src/azmi-commandline/azmi-commandline.csproj
+
+exePath=$(cd ./src/azmi-commandline/bin/Debug/netcoreapp3.0 || exit; pwd)
+PATH="$PATH:$exePath"
+
+echo "azmi getblobs - performance testing, repeat count: 1"
+echo "VM size: $(curl -sS -H Metadata:true "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2018-10-01&format=text")"
+echo "azmi: $(azmi --version)"
+CONT="https://azmitest5.blob.core.windows.net/azmi-pt"
+
+
+
+printf  "\n=================\n"
+echo "azmi listblobs -c $CONT"
```

Comment:
Counter? Took me 2 secs to realize container.

Reply:
Fixed